ui/src/app/guards/sign-in-guard.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import {
+  ActivatedRouteSnapshot,
+  CanActivate,
+  CanActivateChild,
+  Router, RouterStateSnapshot
+} from '@angular/router';
+
+import {Observable} from 'rxjs/Observable';
+
+
+import {SignInService} from 'app/services/sign-in.service';
+
+declare const gapi: any;
```

Comment:
Btw is this declaration needed b/c TypeScript wants it here or is it left over from something else?

Reply:
I believe its needed by typescript/the compiler. It didn't know what gapi was without that.